packages/ui-extensions/src/surfaces/checkout/components/TextArea/examples/basic-textarea.example.html

```diff
@@ -0,0 +1,5 @@
+<s-text-area
+  label="Shipping address"
+  value="1776 Barnes Street, Orlando, FL 32801"
+  rows="3"
+></s-text-area>
```

Comment:
Just curious, could we use the self-closing syntax here?

```suggestion
<s-text-area
  label="Shipping address"
  value="1776 Barnes Street, Orlando, FL 32801"
  rows="3"
/>
```

Reply:
This is mainly to align with admin, so I'll keep it as is